bin/load_shopify.rb

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env ruby --disable=gems
 
-lib_path = File.expand_path("../../../lib", __FILE__)
+lib_path = File.expand_path("../../lib", __FILE__)
```

Comment:
Why did you move load_shopify.rb out of the support directory?

Reply:
It's a stop-gap for now, I want to move it completely because we're ending up with `/usr/local/bin/shopify` and `/usr/local/bin/support/load_shopify.rb`.  Because it's in the bin directory, `gem build` is wrapping it like another executable, which is unnecessary.  I want to put into `lib` but need to test how `shopify` in `/usr/local/bin` can load it (I think the wrapper `gem build` puts around it will take care of it automatically, but need to test).